src/main/java/frc/robot/RobotContainer.java

```diff
@@ -63,9 +69,10 @@ public class RobotContainer
     private final Notepath        _notepath;
     private final ShooterBed      _shooterBed;
     private final ShooterFlywheel _shooterFlywheel;
-    
+    private final Climb           _climb;
+    private Gyro                  _gyro;
```

Comment:
The `Gyro` subsystem should be final as well:

```suggestion
    private final Gyro            _gyro;
```